lib/compiler/passes/generate-bytecode.js

```diff
@@ -70,6 +70,13 @@ const visitor = require("../visitor");
 //
 //        stack.push(input.substring(stack.pop(), currPos));
 //
+// [30] PLUCK
+//
+//        stack.pop(ip + 1);
+//        stack.push(ip + 3);
+//        ...
+//        stack.push(ip + 3 + [ip + 2]);
+//
```

Comment:
That is not the same thing as I described. In the end `stack` should contain one `Array` object with `k` elements, not the `k` elements, as you might think from the current description
```suggestion
// [36] PLUCK n, k, p1, ..., pK
//
//        value = [stack[p1], ..., stack[pK]]; // when k != 1
//        -or-
//        value = stack[p1];                   // when k == 1
//
//        stack.pop(n);
//        stack.push(value);
//
```

Reply:
oh.  heh, i didn't save that buffer.  willfix